content/en/guides/configuration-glossary/configuration-css.md

```diff
@@ -31,3 +31,21 @@ export default {
 ```
 
 Nuxt.js will automatically guess the file type by its extension and use the appropriate pre-processor loader for webpack. You will still need to install the required loader if you need to use them.
+
+### Style Extensions
+
+You can omit the file extension for CSS/SCSS/Postcss/Less/Stylus/... files listed in the css array in your nuxt config file.
+
+```js{}[nuxt.config.js]
+export default {
+  css: ['~/assets/css/main', '~/assets/css/animations']
+}
+```
+
+<base-alert>
+
+If you have two files with the same name eg. main.scss and main.css, and don't add an extension in the css array eg. `css:['~/assets/css/main']`, then the files will be loaded in a specific order, first the css file and then the scss file.
```

Comment:
If I remember correctly, only the css file would be loaded in this case.

Reply:
ok that's not what you said on discord. I shall test it out and let you know what behaviour is

Reply:
ok confirmed. the scss file gets completely ignored. will update. thanks for checking over it while you're on hols

Reply:
ok let me know what you think

Reply:
> ok that's not what you said on discord. I shall test it out and let you know what behaviour is


---

My words were: 
> In such a case, the app.css file would be used as css is before scss in the extension list.

I'd say that sentence explains the behavior. Could've been more explicit on that and mention that *only* this file could've been used though 😋